docs/admin/system_operations/data.md

```diff
@@ -319,7 +322,16 @@ When/How to use recover commands.
         - When some LeoStorage go down during the recover process, just do it over again once the downed nodes come back
         - Generally speaking, it depends heavily on the size of your data and the consistency level(W and D) you chose.
         - The lower consistency level you choose, The more frequently you should run `recover-consistency`.
+- Scale up (Add an additional disk volume on LeoStorage)
+    - Suspend and Stop LeoStorage with `leofs-adm suspend` and `leo_storage stop`
+    - Add an addtional disk volume on the node
+    - Modify leo_storage.conf to append the new disk volume into `obj_containers.num_of_containers`
```

Comment:
> Modify leo_storage.conf to append the new disk volume into `obj_containers.num_of_containers`

Modify leo_storage.conf to append the new disk volume into `obj_containers.path` and also add the number of the containers into `obj_containers.num_of_containers`. _(The number of items of `obj_containers.path` and `obj_containers.num_of_containers` should be the same.)_